engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/privateaccess/PrivateMethodAccessTest.java

```diff
@@ -45,4 +55,90 @@ public void typeDoesNotExposePrivateMethodsToPolyglot() {
       assertThat("public method is exposed to polyglot", pubMethod.canExecute(), is(true));
     }
   }
+
+  @Test
+  public void canCallPrivateMethod_AsCallback() throws IOException {
+    var libDir = tempFolder.newFolder("Lib").toPath();
+    ProjectUtils.createProject(
+        "Lib",
+        Set.of(
+            new SourceModule(
+                QualifiedName.fromString("Mod"),
+                """
+                private priv_method x = x
```

Comment:
There is no new test for `.name` syntax. Following should probably be also tested:
```
private 

type A
   V v
 
   private value self = self.v
```
and then get `A.V 42` and call `.value` on it from another project

Reply:
Sufficient. But: Where has the attitude to write unit tests for everything gone?